### ISSUE TITLE
Needs installation tutorial

### ISSUE THREAD

**Issue by ihatecsv** [jaseg/ffi#1]
How do you expect us to use this if there's no installation tutorial and example usage?!

Sheesh.

**Comment by jaseg** [comment]
Hahahahahahahahahaha hahahaha haha hahahaha! :laughing: :laughing: :laughing:

**Comment by jaseg** [comment]
No, seriously, if you manage to make it work, tell me how you did it, because I only have that one VM where I know it works built over several weeks, and I have no idea how to put together another one. Some stuff (mostly mono and go) needs to be built from source to get the most recent bugfixes, but I think I noted _that_ down in the README.
